### ISSUE TITLE
Rewrite commands

### ISSUE THREAD

**Issue by Rexogamer** [Cytrus-RE/cytrus-re#1]
We basically start anew with the commands.

**Comment by Rexogamer** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [rewrite commands part 1/4](https://trello.com/c/oADR0E5Y/5-rewrite-commands-part-1-4)

**Comment by DestroyedEDGE** [comment]
i mean now we kinda have to considering how much djs v12 breaks things

**Comment by Oxygemo** [comment]
> i mean now we kinda have to considering how much djs v12 breaks things

no, the way we updated was just changing things. dont have to make the command a new

**Comment by Rexogamer** [comment]
> > i mean now we kinda have to considering how much djs v12 breaks things
> 
> no, the way we updated was just changing things. dont have to make the command a new

we should still probably rewrite some of them

**Comment by CommandBlock6417** [comment]
Test them one by one, with different scenarios and even deliberately cause it to error and see what is dead and needs fixing.
